dev/MenuFlyout/MenuFlyout_rs1_themeresources.xaml

```diff
@@ -24,7 +24,7 @@
             <StaticResource x:Key="MenuFlyoutSubItemBackground" ResourceKey="SystemControlTransparentBrush" />
             <StaticResource x:Key="MenuFlyoutSubItemBackgroundPointerOver" ResourceKey="SystemControlHighlightListAccentMediumBrush" />
             <StaticResource x:Key="MenuFlyoutSubItemBackgroundPressed" ResourceKey="SystemControlHighlightListAccentHighBrush" />
-            <StaticResource x:Key="MenuFlyoutSubItemBackgroundSubMenuOpened" ResourceKey="SystemControlHighlightListAccentLowBrush" />
+            <StaticResource x:Key="MenuFlyoutSubItemBackgroundSubMenuOpened" ResourceKey="SystemControlHighlightListLowRevealBackgroundBrush" />
```

Comment:
I just recalled MenuFlyout reveal was just added since rs4, this reveal brush worked in lower versions because it has a fallback value. You might consider change it to something like `SystemControlHighlightListLowBrush` instead

Reply:
@kikisaints , can you please look at to make sure the brush being picked here is appropriate?

Reply:
@kaiguo and @chigy
This looks like the correct translation from solid color to Reveal, however this brush is typically used for PointerOver backgrounds. What state is this SubMenuOpened in?

Reply:
Is there, or should there be a difference between the Pointer Over visual state or the Selected MenuFlyoutItem state?

Reply:
@kikisaints the spec says "use list hover for menu selected", so I think that's the right brush to use?

Reply:
Then that's the right one, it sounds like.

Reply:
@kaiguo I just tested it in an RS3 VM, it seems to work fine although, the hover color in RS3 is still the accent color although, the hover in RS4 is the gray hover color, so I might need to change `MenuFlyoutSubItemBackgroundPointerOver` also.

@kikisaints SubMenuOpened is it's own state for when the sub menu is revealed.

Reply:
@chigy if RS3 hover is using accent color, should we just keep SubMenuOpened color unchanged on rs3 since they are already the same? 

Or do we change both SubMenuOpend and PointerOver to SystemControlHighlightListLowBrush like @savatia said?

Reply:
@kaiguo , let me make sure I understand this correctly...
Options for RS3 you are proposing are:

- Option 1 (Hover - Accent, SubMenuOpened - Accent)
- Option 2 (Hover - Grey, SubMenuOpened - Grey)

If there is no back compat concerns, I feel doing option 2 will keep us sane that we don't have to keep track of what's different and remember why they are different? But I don't know what RS3 rules are... Is the rule to keep it as consistent to what was before or are we allowed to change freely?

Reply:
I didn't find any rule in the spec, i'm leaning towards option 2 as well, that will give us a consistent looking across all versions, so let's make both grey if no one opposes @savatia

Reply:
Done